examples/jsm/nodes/math/MathNode.js

```diff
@@ -152,7 +151,7 @@ class MathNode extends TempNode {
 
 			return new MathNode( MathNode.NORMALIZE, mulNode ).build( builder );
 
-		} else if ( method === MathNode.SATURATE ) {
+		} else if ( method === MathNode.CLAMP && b === null ) {
```

Comment:
Maybe this should be done rather in `ShaderNodeBaseElements`? I.e. `clamp` will have two optional parameters that default to 0 and 1?